thirdparty_build.gradle

```diff
@@ -9,8 +9,25 @@ buildscript {
 /*DJVU CHAIN*/
 task downloadDjvu {
     doLast {
-        exec {
-            commandLine = ['git', 'clone', 'git://git.code.sf.net/p/djvu/djvulibre-git', djvu]
+        def djvuFolder = new File(djvu)
+        if (djvuFolder.exists()) {
+            // Clean previous checkout
+            exec {
+                commandLine = ['git', '-C', djvu, 'checkout', '.']
+            }
+            // Switch to master
+            exec {
+                commandLine = ['git', '-C', djvu, 'checkout', 'master']
+            }
+            // Pull
+            exec {
+                commandLine = ['git', '-C', djvu, 'pull']
```

Comment:
Probably simple `fetch` would be enough instead of `pull`

Reply:
Fixed